iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/TerminalRefSerDe.java

```diff
@@ -79,16 +79,20 @@ public static Terminal readTerminal(NetworkDeserializerContext context, Network
         return TerminalRefSerDe.resolve(id, side, n);
     }
 
-    public static void readTerminalRef(NetworkDeserializerContext context, Network network, Consumer<Terminal> endTaskTerminalConsumer) {
+    public static void readTerminalRefAttributes(NetworkDeserializerContext context, Network network, Consumer<Terminal> endTaskTerminalConsumer) {
```

Comment:
Thanks to your comment I understood that this method was not really needed. 
I made a small refactor that deletes it.
see 993baaea9c0647aabac94c56e55b720a54ab5582

Reply:
You are right, it is indeed much better like this.

Just one thing: before this PR,
`context.getReader().readEndNode();`
was before
` context.getEndTasks().add(() -> ...`
in `TerminalRefSerDe.readTerminalRef`. Do you need to move it or could it be restored at its previous place?

If it could be moved, maybe could you also move it in `BoundaryRefSerDe.readBoundaryRef` to be consistent.

Reply:
It can be restored, you are rigth.
fixed in f6a9fbf0fc509b121c08b4005ad4e2cbac251fed